views/components/navbar.njk

```diff
@@ -6,9 +6,11 @@
       <span class="navbar-toggler-icon"></span>
     </button>
     <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
+      {% if role == '2' %}
```

Comment:
Should this be if role == 2 and 1, so an admin can still see the 'view job roles' button on the navbar